.circleci/config.yml

```diff
@@ -174,6 +174,15 @@ jobs:
     steps:
       - checkout
 
+      - restore_cache:
+          key: bundle-v1-{{ checksum "ios/Gemfile.lock" }}-{{ arch }}
```

Comment:
Does `name` property works?
It would be better if it was `Restoring npm cache`, `Restoring gems cache`, etc

Reply:
not sure, haven't seen it in docs, but can try

Reply:
Can you do this for all similar tasks?
I can see there are two confusing tasks on android, for example:
![image](https://user-images.githubusercontent.com/804994/60613394-2bb3a600-9da1-11e9-8576-ec191cd5d1d7.png)

Reply:
added names